src/workers/cdc.ts

```diff
@@ -173,8 +174,17 @@ const onUserChange = async (
       await notifyDevCardEligible(logger, data.payload.after.id);
     }
 
-    if (data.payload.after.reputation >= submissionAccessThreshold) {
-      await notifySubmissionGrantedAccess(logger, data.payload.after.id);
+    if (
+      data.payload.after.reputation >= submissionAccessThreshold &&
+      data.payload.before.reputation < submissionAccessThreshold
+    ) {
+      try {
+        await con.getRepository(UserState).insert({
+          userId: data.payload.after.id,
+          key: UserStateKey.CommunityLinkAccess,
+          value: true,
+        });
+      } catch (ex) {}
```

Comment:
That's not a good practice to ignore every error. We want to ignore only the duplicate violation error, otherwise throw the error